eq-author/src/App/page/Logic/Routing/RoutingPage/NoRouting.js

```diff
@@ -33,34 +36,47 @@ const Paragraph = styled.p`
 
 const AddRoutingButton = styled(Button)`
   margin: 2em auto 1em;
+  ${props =>
+    props.isLastPage &&
+    `
+     pointer-events: none;
+     opacity: 0.6;`};
 `;
 
-const RoutingRuleSetMsg = ({
-  title,
-  onAddRouting,
-  children,
-  ...otherProps
-}) => (
-  <Container {...otherProps}>
-    <Icon />
-    <Title>{title}</Title>
-    <Paragraph>{children}</Paragraph>
-    <AddRoutingButton
-      small
-      naked
-      variant="primary"
-      onClick={onAddRouting}
-      data-test="btn-add-routing"
-    >
-      <IconText icon={IconAddRule}>Add your first rule</IconText>
-    </AddRoutingButton>
-  </Container>
-);
+const RoutingRuleSetMsg = ({ onAddRouting, page, ...otherProps }) => {
+  let lastSection, lastSectionPage, lastPageId;
+  if (otherProps.questionnaire) {
+    const questionnaire = otherProps.questionnaire;
+    lastSection = questionnaire.sections.length;
+    lastSectionPage = questionnaire.sections[lastSection - 1].pages.length;
+    lastPageId = questionnaire.sections[lastSection - 1].pages[lastSectionPage - 1].id;
```

Comment:
To increase readability, I'm suggesting renaming the variables as I feel their current names are slightly misleading.

```suggestion
  let nSections, nLastSectionPages, lastPageId;
  if (otherProps.questionnaire) {
    const questionnaire = otherProps.questionnaire;
    nSections = questionnaire.sections.length;
    nLastSectionPages = questionnaire.sections[nSections - 1].pages.length;
    lastPageId = questionnaire.sections[nSections - 1].pages[nLastSectionPages - 1].id;
```